### ISSUE TITLE
mdquiz reports are showing "alt text" and empty spaces instead of images

### ISSUE THREAD

**Issue by flannery-denny** [bootstrapworld/curriculum#2876]
Here's what  I'm seeing for dot plots to histograms.
<img width="412" height="580" alt="Image" src="https://github.com/user-attachments/assets/69daf188-babe-44d3-92e3-21717dbf37e9" />

I am also empty spaces where the images belong for all questions on my Histograms, Visualizing Shape - Cumulative (Results) ∙ Mon 8/31
1. I should be seeing images
2. When the images don't load, I should be seeing something other than the words "alt text"

What can FD do to support #2?

**Comment by ds26gte** [comment]
The quiz.json contains lines like
```
            {
              "id": "dpb",
              "content": "![alt text](./dot-plot-b.png 'some text')"
            },
```
The `alt text` is coming from here. You can use more descriptive text here.

(The bug has been fixed, and the proper images should show now.)

**Comment by flannery-denny** [comment]
@ds26gte in the reports I received at 9pm... 

- images displayed correctly for 📊 Histograms, Visualizing Shape - Cumulative (Results) ∙ fd87
- images were missing for 📊 Dot Plots to Histograms (Results) ∙ fd - monday 8/31

**Comment by flannery-denny** [comment]
@schanzer The 4:34 report for Dot Plots to Histograms had images that loaded correctly. And the 9pm report did not. Any chance that's related to the work you've done this evening?  @ds26gte thought he'd fixed the bug...

**Comment by schanzer** [comment]
@ds26gte , did you commit your changes to the `db` repo after fixing? I'm not seeing them there...
